homeworks/karolinavalko/4_exceptions_logging/to_do.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import logging.config
+import os
+
+
+print(os.getcwd())
+
+logging.config.fileConfig(
```

Comment:
ugyanazok a dolgok igazak itt is, amit a json-ös megoldáshoz írtam (relative path, nem kell raw string, forward slash-ek)